apps/src/code-studio/components/progress/ProgressDot.jsx

```diff
@@ -119,7 +122,17 @@ const styles = {
       fontSize: 16,
       lineHeight: '32px'
     },
-    icon: progressStyles.dotIcon,
```

Comment:
This style was once shared between multiple places, but is only used here now, so I pulled it back out of the shared file